org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -328,6 +328,83 @@
       </snippet>
    </extension>
 
+   <!-- XML Language -->   
+   <extension
+         point="org.eclipse.core.contenttype.contentTypes">
+      <content-type
```

Comment:
Please add DTD content type too that it i ssupported by lsp4xml.

Reply:
Let's make it a separate issue after this patch is merged.